service/gcs/main.go

```diff
@@ -20,6 +21,45 @@ import (
 	"go.opencensus.io/trace"
 )
 
+func kmsgLoop() {
+	f, err := os.Open("/dev/kmsg")
+	if err != nil {
+		logrus.WithError(err).Error("failed to open kmsg")
+		return
+	}
+	// /dev/kmsg never returns EOF, so we just sit in the loop waiting for more
+	// log entries forever.
+	for {
+		// Buffer size from CONSOLE_EXT_LOG_MAX	in Linux's printk.h.
+		b := [8192]byte{}
+		n, err := f.Read(b[:])
```

Comment:
why is the b[:] necessary? can you not just pass through b here? Was this a style choice?

Reply:
Talked to Kevin offline, this converts an array into a slice.